### PR TITLE
Add new command "pano field scaffold"

### PR DESCRIPTION
**Pull request by mrknmc**
Creates field files for any slug that is referenced in a model but doesn't have a field file.

### CODE REVIEW

File: src/panoramic/cli/command.py [hunk 1]
```diff
@@ -369,18 +370,55 @@ def delete_orphaned_fields(target_dataset: Optional[str] = None, yes: bool = Fal
...
+            # Add creation action
+            action_list.add_action(Action(current=None, desired=map_error_to_field(error)))
+
+    if action_list.is_empty:
+        echo_info('No issues found')
+        return
+
+    if not yes and not click.confirm(
+        'You need to define these fields before they can be queried. Do you want to create them?'
+    ):
```
Comment (mrknmc) [inline src/panoramic/cli/command.py:421]: Output:

```
> pano field scaffold
Loading local state...
Fields referenced in models without definition in dataset facebook_ads:
  like
  link_click
  omni_purchase
  onsite_conversion_messaging_block
You need to define these fields before they can be queried. Do you want to create them? [y/N]:
```

File: src/panoramic/cli/command.py [hunk 2]
```diff
@@ -369,18 +370,55 @@ def delete_orphaned_fields(target_dataset: Optional[str] = None, yes: bool = Fal
...
+            echo_info(f'  {error.field_slug}')
             # Add deletion action
             action_list.add_action(Action(current=fields_by_slug[error.field_slug], desired=None))
 
     if action_list.is_empty:
         echo_info('No issues found')
         return
 
-    if not yes and not click.confirm('Do you want to remove offending fields?'):
+    if not yes and not click.confirm('You will not be able to query these fields. Do you want to remove them?'):
```
Comment (mrknmc) [inline src/panoramic/cli/command.py:385]: Final output:

```> pano field cleanup
Loading local state...
Fields without calculation or reference in a model in dataset pinterest:
  privacy
  value
  currency
  name
  reason
  status
  end_time
  country
  layout
  schema
  id
You will not be able to query these fields. Do you want to remove them? [y/N]:
```

Comment (mrknmc) [reply src/panoramic/cli/command.py:386]: @kubamahnert

Comment (kubamahnert) [reply src/panoramic/cli/command.py:386]: 👍  looks great

**Review by raethlo — APPROVED** [review]
Nice 🍫

**Pull request merged**